terracumber_config/tf_files/Uyuni-Master-PRV.tf

```diff
@@ -148,6 +148,7 @@ module "cucumber_testsuite" {
       }
       additional_packages = [ "venv-salt-minion" ]
       install_salt_bundle = true
+      image = "sles15sp4o"
```

Comment:
Shouldn't Uyuni Proxy Non-containerized use Leap 15.5 as Base OS?

Reply:
You infact wants to change this line https://github.com/SUSE/susemanager-ci/blob/master/terracumber_config/tf_files/tfvars/PR-testing-uyuni.tfvars#L5

Reply:
Probably... but for now I have a weird issue with it and it's the containerized proxy that causes me troubles